node/src/Commands.ts

```diff
@@ -3573,3 +3573,56 @@ export function createBZPopMin(
 ): command_request.Command {
     return createCommand(RequestType.BZPopMin, [...keys, timeout.toString()]);
 }
+
+/**
+ * Optional arguments to getex command.
```

Comment:
```suggestion
 * Time unit representation which is used in optional arguments for {@link BaseClient.getex|getex} and {@link BaseClient.set|set} command.
```

Reply:
Please update `SetOptions` and corresponding docs (examples) and tests to use this enum